http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcJsonConfiguration.java

```diff
@@ -152,13 +157,21 @@ public class OidcJsonConfiguration {
     protected int tokenMinimumTimeToLive = 0;
     @JsonProperty(MIN_TIME_BETWEEN_JWKS_REQUESTS)
     protected int minTimeBetweenJwksRequests = 10;
+    @JsonProperty(POST_LOGOUT_URI)
+    protected String postLogoutUri;
     @JsonProperty(PUBLIC_KEY_CACHE_TTL)
     protected int publicKeyCacheTtl = 86400; // 1 day
     // https://tools.ietf.org/html/rfc7636
     @JsonProperty(ENABLE_PKCE)
     protected boolean pkce = false;
     @JsonProperty(IGNORE_OAUTH_QUERY_PARAMETER)
     protected boolean ignoreOAuthQueryParameter = false;
+    @JsonProperty(LOGOUT_PATH)
+    protected String logoutPath;
+    @JsonProperty(LOGOUT_CALLBACK_PATH)
+    protected String logoutCallbackPath;
+    @JsonProperty(LOGOUT_SESSION_REQUIRED)
+    protected boolean logoutSessionRequired = true;
     @JsonProperty(VERIFY_TOKEN_AUDIENCE)
```

Comment:
I think we're missing something for what's currently called `logoutSessionWaitingLimit` in this file.

Reply:
This is an Elytron specific setting.  There is no mention of this 
and any of the specs.   Still add it to OidcJsonConfiguration and
other structures as needed?

Reply:
Since it's part of the OidcClientConfiguration, it would need to be present here. Maybe check with @pedroigor to see if this is something that should be configurable or not.